samba-native-php7.3/smbclient.ini

```
@@ -0,0 +1 @@
+extension="smbclient.so"
```

```suggestion
extension=smbclient
```
I think they're usually written without the extension and without quotes :)

---

No idea, I just copy pasted from the [samba-native-php7.1 files](https://github.com/nextcloud/docker-ci/blob/322c63de48793d4e1b358f893180123ade1fcdc3/samba-native-php7.1/smbclient.ini) :man_shrugging: 

I have checked how they appear in other ini files in the PHP 7.3 Docker image and this is the result:
```
...]# docker run --rm nextcloudci/php7.3:php7.3-3 find /etc/php/7.3/cli/conf.d/ -type f -exec grep -Hn extension {} \;
/etc/php/7.3/cli/conf.d/20-ast.ini:1:extension=ast.so
```

Not a very big sample, unfortunately :-P So as you wish, no quotes and no extension, no quotes but extension, quotes and extension... I have no preference ;-)

---

I really do not care xD
If it works, good for me :D